business_requirement/views/business_view.xml

```diff
@@ -0,0 +1,351 @@
+                        clickable="True"
+                        statusbar_visible="draft,confirmed,approved,in_progress,done,cancel,drop"
+                    />
+                </header>
+                <sheet>
+                    <div
+                        class="oe_button_box"
+                        name="button_box"
+                        groups="base.group_user"
+                    >
+                        <button
+                            class="oe_stat_button"
+                            name="portal_publish_button"
+                            type="object"
+                            icon="fa-globe"
+                        >
+                            <label for="portal_published" />
+                            <field name="portal_published" widget="toggle_button" />
+                        </button>
+                    </div>
+                    <div class="oe_left">
+                        <field
+                            name="kanban_state"
+                            class="oe_inline"
+                            widget="kanban_state_selection"
+                        />
+                    </div>
+                    <h1>
+                        <field name="name" class="oe_inline" readonly="1" /> -
+                        <field name="description" default_focus="1" />
+                    </h1>
+                    <group col="4">
+                        <field name="responsible_user_id" required="1" />
+                        <field name="partner_id" />
+                        <field name="origin" />
+                        <field name="category_ids" widget="many2many_tags" />
+                        <field name="to_be_reviewed" />
+                        <field name="reviewer_ids" widget="many2many_tags" />
+                        <field
+                            name="priority"
+                            groups="base.group_user"
+                            widget="priority"
+                        />
+                    </group>
+                    <notebook>
+                        <page string="Business Requirement">
+                            <group string="Stakeholder Story">
+                                <field name="business_requirement" nolabel="1" />
+                            </group>
+                            <group string="Scenario">
+                                <field name="scenario" nolabel="1" />
+                            </group>
+                            <group string="Gap">
+                                <field name="gap" nolabel="1" />
+                            </group>
+                            <group string="Test Case">
+                                <field name="test_case" nolabel="1" />
+                            </group>
+                            <group string="Terms and Conditions">
+                                <field name="terms_and_conditions" nolabel="1" />
+                            </group>
```

Comment:
```suggestion
                            <separator string="Stakeholder Story"/>
                            <field name="business_requirement"/>
                            <separator string="Scenario"/>
                            <field name="scenario"/>
                            <separator string="Gap"/>
                            <field name="gap"/>
                            <separator string="Test Case"/>
                            <field name="test_case"/>
                            <separator string="Terms and Conditions"/>
                            <field name="terms_and_conditions"/>
```

In this version, html fields have a strange behavior when they are inside a group without a label. I suggest replacing the groups with separators so that it looks similar to the previous version. Thanks for the contribution

Reply:
```suggestion
                            <group string="Stakeholder Story">
                                <field name="business_requirement" nolabel="1" colspan="2"/>
                            </group>
                            <group string="Scenario">
                                <field name="scenario" nolabel="1" colspan="2"/>
                            </group>
                            <group string="Gap">
                                <field name="gap" nolabel="1" colspan="2"/>
                            </group>
                            <group string="Test Case">
                                <field name="test_case" nolabel="1" colspan="2"/>
                            </group>
                            <group string="Terms and Conditions">
                                <field name="terms_and_conditions" nolabel="1" colspan="2"/>
                            </group>
```
Sorry, I just found that it can be done adding `colspan="2"`